README.md

```diff
@@ -129,10 +144,6 @@ GITHUB_TOKEN=<*> npm run data:update
 
 ```sh
 # once environment variables are configured, install vercel and deploy
-npm i -g vercel
+yarn global add vercel
```

Comment:
these days i default to just explaining npm, because if someone is using yarn they know what the yarn equivalent is, but that may not be true the other way around (and explaining both seems like too much text).

that said, we probably want people to use yarn for this repo because of our lockfile, so maybe it makes sense to just say yarn here as well.